logfire/_internal/main.py

```diff
@@ -150,9 +154,9 @@ def _logs_tracer(self) -> Tracer:
     def _spans_tracer(self) -> Tracer:
         return self._get_tracer(is_span_tracer=True)
 
-    def _get_tracer(self, *, is_span_tracer: bool, otel_scope: str | None = None) -> Tracer:  # pragma: no cover
+    def _get_tracer(self, *, is_span_tracer: bool) -> Tracer:  # pragma: no cover
```

Comment:
Unrelated, I just noticed that the `otel_scope` argument was unused which was confusing.